app/scripts/models/date.coffee

```diff
@@ -0,0 +1,38 @@
+root = exports ? this
+
+root.Date = class Date
```

Comment:
so I think date is a bit of a problematic class, for a few reasons:

1. There is already a natural Date class in the browser, so yours should be called GameDate or some such.
2. The idea of getting the "next" date (which may change phase or season or year or..?) seems a little silly. I propose those levers be separate and less generalized. On the other hand, if you just rename `Date` into `Phase`, I might drop this concern.
3. What do you think of making a global for defs? it seems like they'll be passed into every single class in the whole game...
4. We are building a Diplomacy game. It might make sense to make it as non-generic as possible at first, for simplicity's sake. I'd bring the ordering of phases right into this class instead of a separate definitions file. If we want to build Risk on the same engine in the future, we can abstract the commonalities at that point...